src/Gui/Splashscreen.h

```diff
@@ -66,7 +76,7 @@ class GuiExport AboutDialogFactory
     static void setDefaultFactory(AboutDialogFactory *factory);
 
 private:
-    static AboutDialogFactory* factory;
+    static AboutDialogFactory* factory {};

```

Comment:
That is not allowed. A static member variable must be initialized in the source file the way how it was before.